pkg/backup/backup_job.go

```diff
@@ -148,46 +148,12 @@ func backup(
 		execCtx.ExecCfg().InternalDB,
 		execCtx.User(),
 	)
-	// TODO(benesch): verify these files, rather than accepting them as truth
-	// blindly.
-	// No concurrency yet, so these assignments are safe.
-	iterFactory := backupinfo.NewIterFactory(backupManifest, defaultStore, encryption, &kmsEnv)
-	it, err := iterFactory.NewFileIter(ctx)
+	completedSpans, completedIntroducedSpans, err := getCompletedSpans(
```

Comment:
looks like there's  `completedSpans` and `completededIntroducedSpans` instantiations above. no need to.